deenurp/subcommands/rdp_extract_genbank.py

```diff
@@ -106,7 +125,7 @@ def action(a):
         if a.header:
             header = ('version', 'seqname', 'tax_id', 'accession',
                       'description', 'length', 'ambig_count',
-                      'is_type', 'rdp_lineage')
+                      'is_type', 'rdp_lineage', 'is_classified')
```

Comment:
Let's change 'is_classified' to 'taxid_is_classified' to clarify that we are just looking at the tax_id here and not other elements of the the sequence record. This should also be clarified in the module's docstring.